test/forkMainnet/TokenCollector.t.sol

```diff
@@ -120,7 +120,11 @@ contract TestTokenCollector is Addresses {
         bytes32 r,
         bytes32 s
     ) private pure returns (bytes memory) {
-        return abi.encode(TokenCollector.Source.Token, abi.encode(permit.owner, permit.spender, permit.amount, permit.deadline, v, r, s));
+        return
+            bytes.concat(
+                abi.encodePacked(TokenCollector.Source.TokenPermit),
+                abi.encode(permit.owner, permit.spender, permit.amount, permit.deadline, v, r, s)
```

Comment:
Q: Cannot just `abi.encodePacked(TokenPermit, abi.encode( ... ))`?

Reply:
It works as well. Adjusted in 4b13f26.

Reply:
I think we could get rid of `bytes.concat` also?

Reply:
Yeah, fixed in 0f31586